frontend/awx/AwxMain.tsx

```diff
@@ -30,6 +30,7 @@ export default function AwxMain() {
       }
       masthead={<AwxMasthead />}
       navigation={navigation}
+      basename={process.env.AWX_ROUTE_PREFIX}
```

Comment:
Do we need a fallback here?

Reply:
should not need a fallback. if empty it will just not a a basename which is the desired functionality